Dockerfile

```diff
@@ -57,8 +58,9 @@ RUN --mount=type=bind,source=.,target=.,readonly\
 
 
 FROM debian:bookworm-slim AS wardenkms
-COPY --from=wardenkms-build /build/wardenkms /
-ADD --checksum=sha256:b0c3b761e5f00e45bdafebcfe9c03bd703b88b3f535c944ca8e27ef9b891cd10 https://github.com/CosmWasm/wasmvm/releases/download/v1.5.2/libwasmvm.x86_64.so /lib/libwasmvm.x86_64.so
+COPY --chown=nobody:nogroup --from=wardenkms-build /build/wardenkms /
+ADD --chown=nobody:nogroup --checksum=sha256:b0c3b761e5f00e45bdafebcfe9c03bd703b88b3f535c944ca8e27ef9b891cd10 https://github.com/CosmWasm/wasmvm/releases/download/v1.5.2/libwasmvm.x86_64.so /lib/libwasmvm.x86_64.so
+USER nobody
```

Comment:
Why not state the USER on top and then add and download everything with that?
I think then the files would inherit the `user`

Reply:
Unfortunately, that's way doesn't work.
USER works in run context, not build context